### PR TITLE
Use application/applications instead of app/apps

### PR DESCRIPTION
**Pull request by alan-agius4**
We should prefer to use the full word and use application instead of app.

### CODE REVIEW

File: aio/content/examples/upgrade-phonecat-1-typescript/README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Start like any example
 
     npm run start
 
-You'll find the app under the /app path: http://localhost:3002/app/index.html
+You'll find the application under the /application path: http://localhost:3002/app/index.html
```
Comment (clydin) [inline aio/content/examples/upgrade-phonecat-1-typescript/README.md:24]:
```suggestion
You'll find the application under the /app path: http://localhost:3002/app/index.html
```

File: aio/content/guide/app-shell.md [hunk 2]
```diff
@@ -11,14 +11,14 @@ Learn more in [The App Shell Model](https://developers.google.com/web/fundamenta
 
 You can do this with the following CLI command:
 <code-example language="bash">
-ng new my-app --routing
+ng new my-application --routing
```
Comment (clydin) [inline aio/content/guide/app-shell.md:14]:
```suggestion
ng new my-app --routing
```
The remainder of the guide references `my-app`.

File: aio/content/guide/browser-support.md [hunk 3]
```diff
@@ -393,7 +393,7 @@ The following polyfills are used to test the framework itself. They are a good s
 
 
 \* Figures are for minified and gzipped code,
-computed with the <a href="http://closure-compiler.appspot.com/home">closure compiler</a>.
+computed with the <a href="http://closure-compiler.applicationspot.com/home">closure compiler</a>.
```
Comment (clydin) [inline aio/content/guide/browser-support.md:396]:
```suggestion
computed with the <a href="http://closure-compiler.appspot.com/home">closure compiler</a>.
```

File: aio/content/guide/service-worker-intro.md [hunk 4]
```diff
@@ -1,12 +1,12 @@
 # Angular service worker introduction
 
-Service workers augment the traditional web deployment model and empower applications to deliver a user experience with the reliability and performance on par with natively-installed code. Adding a service worker to an Angular application is one of the steps for turning an application into a [Progressive Web App](https://developers.google.com/web/progressive-web-apps/) (also known as a PWA).
+Service workers augment the traditional web deployment model and empower applications to deliver a user experience with the reliability and performance on par with natively-installed code. Adding a service worker to an Angular application is one of the steps for turning an application into a [Progressive Web App](https://developers.google.com/web/progressive-web-applications/) (also known as a PWA).
```
Comment (clydin) [inline aio/content/guide/service-worker-intro.md:3]:
```suggestion
Service workers augment the traditional web deployment model and empower applications to deliver a user experience with the reliability and performance on par with natively-installed code. Adding a service worker to an Angular application is one of the steps for turning an application into a [Progressive Web App](https://developers.google.com/web/progressive-web-apps/) (also known as a PWA).
```

File: aio/content/guide/router.md [hunk 5]
```diff
@@ -23,21 +23,21 @@ For a more in-depth experience of building an Angular app, see the [Tour of Hero
...
+## Generate an application with routing enabled
 
-The following command uses the Angular CLI to generate a basic Angular app with an app routing module, called `AppRoutingModule`, which is an NgModule where you can configure your routes.
-The app name in the following example is `routing-app`.
+The following command uses the Angular CLI to generate a basic Angular application with an application routing module, called `AppRoutingModule`, which is an NgModule where you can configure your routes.
+The application name in the following example is `routing-app`.
 
 <code-example language="none" class="code-shell">
-  ng new routing-app --routing
+  ng new routing-application --routing
```
Comment (clydin) [inline aio/content/guide/router.md:32]:
```suggestion
  ng new routing-app --routing
```

**Review by clydin — CHANGES_REQUESTED** [review]

**Comment by jelbourn** [comment]
I'm not sure this is a change worth making. While I agree that abbreviations are better avoided, I think that this PR introduces a level of churn I'd rather avoid. The term "app" is also quite commonly used compared to other abbreviations, to the point where I'd just call it its own word.

**Comment by petebacondarwin** [comment]
Screenshot 2020-10-13 at 19 43 48

**Comment by alan-agius4** [comment]
Fair enough.

**Pull request closed without merging**